src/generator/text-to-state-machine.js

```diff
@@ -18,7 +18,23 @@ export default (code) => {
     const {results} = parser;
     const [result] = results;
 
-    const {states} = result;
+    const {states, timers} = result;
+
+    // Convert all integer states to int
+    states.forEach(({state}, index) => {
```

Comment:
If we convert strings to numbers here, how is the next step in the pipeline able to use the strings?

Reply:
We only convert strings that are convertible to a numbers (the isNaN function check this), all the other strings are kept as strings